SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -55,6 +79,9 @@ struct FConnectionConfig
 	Worker_NetworkConnectionType LinkProtocol;
 	Worker_ConnectionParameters ConnectionParams;
 	uint8 TcpMultiplexLevel;
+	uint8 TcpNoDelay;
```

Comment:
It looks like a `bool` to me

Reply:
Kept it an uint as that's what the worker config wants